src/wwwroot/app/views/settings/members/members.controller.js

```diff
@@ -27,7 +27,6 @@ export default function MembersController($scope, $element, SettingsService, Use
    function _initUsers() {
       UserService.getUsers().then((result) => {
          vm.users = result;
```

Comment:
may be so,ething like this:

UserService.getUsers().then(result => set(vm, 'users', result));

or:

let curriedSet = curry(set, 3);
...
UserService.getUsers().then(curriedSet(vm, 'users'));